src/Components/Components/src/RenderTree/Renderer.cs

```diff
@@ -653,6 +711,21 @@ private void RenderInExistingBatch(RenderQueueEntry renderQueueEntry)
             }
         }
 
+        private static void ProcessAggregateException(List<Exception> exceptions, AggregateException aggregateException)
+        {
+            var innerExceptions = aggregateException.Flatten().InnerExceptions;
+            for (var i = 0; i < innerExceptions.Count; i++)
+            {
+                var exception = exceptions[i];
+                if (exception is TaskCanceledException)
```

Comment:
Just to double check: do you want TaskCanceledException or OperationCanceledException (TCE's base type) here?

Reply:
Good point. But if we're no longer filtering out cancellation, lines 683-687 can be removed anyway.

Reply:
Maybe the whole thing can just be `exceptions.AddRange(aggregateException.Flatten().InnerExceptions)`.

Reply:
I took the entire thing away, no need to flatten, the error handlers will do that for you, so this just became exception.Add